pkg/types/types.go

```diff
@@ -36,13 +36,23 @@ const (
 	EngineFrontendISCSI    = "tgt-iscsi"
 )
 
+type ReaderWriterUnmapperAt interface {
+	io.ReaderAt
+	io.WriterAt
+	UnmapperAt
+}
+
 type ReaderWriterAt interface {
 	io.ReaderAt
 	io.WriterAt
 }
```

Comment:
`ReaderWriterAt` is not useful. Why not just add `UnmapperAt` in the original `ReaderWriterAt` interface?

Reply:
Sure. we can directly modify `ReaderWriterAt` to `ReaderWriterUnmapperAt`